Client.xcodeproj/project.pbxproj

```diff
@@ -18225,7 +18225,7 @@
 					"-ObjC",
 					"-lxml2",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = org.mozilla.ios.FennecEnterprise;
```

Comment:
This change is not correct. The xctrunner should only be appended to the XCUITests bundle id